x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_hosts_table.tsx

```diff
@@ -100,11 +105,15 @@ const sortValues = (aValue: any, bValue: any, { direction }: Sorting) => {
 };
 
 const sortTableData =
-  ({ direction, field }: Sorting) =>
+  ({ direction, field }: Sorting, displayAlerts: boolean) =>
   (a: HostNodeRow, b: HostNodeRow) => {
     const aValue = a[field as keyof HostNodeRow];
     const bValue = b[field as keyof HostNodeRow];
 
+    if (displayAlerts) {
+      return sortValues(aValue, bValue, { direction, field });
+    }
```

Comment:
Q: `field` already contains the table attribute that needs to be sorted for, why do we need to pass the `displayAlert`?

Reply:
Correct, that was a leftover from testing the sorting, not needed. Thanks for the catch